toolchain/check/context.h

```diff
@@ -89,7 +89,8 @@ class Context {
 
   // TODO: Remove this and pass the C++ context to the constructor.
   auto set_cpp_context(std::unique_ptr<CppContext> cpp_context) {
-    CARBON_CHECK(cpp_context, "C++ context set more than once");
+    CARBON_CHECK(!cpp_context_ || !cpp_context,
+                 "C++ context set more than once");
```

Comment:
```suggestion
                 "Already have a C++ context");
```

The message sounds odd to me because this is making valid:

```
  set_cpp_context(foo);
  set_cpp_context(nullptr);
  set_cpp_context(bar);
```

Of course the prior message I'm realizing was also wrong, since it was actually requiring to be called without nullptr. I admit this is a nit though so feel free to ignore it (or just remove the message, and I probably wouldn't have commented).